pom.xml

```diff
@@ -226,6 +226,9 @@
                   <exclude>**/test/resources/**/*.txt</exclude>
                   <exclude>**/test/**/.placeholder</exclude>
                   <exclude>.repository/**/*</exclude>
+                  <exclude>**/.tox/**/*</exclude>
+                  <exclude>**/*.egg-info/**</exclude>
+                  <exclude>**/.eggs/**</exclude>
```

Comment:
please put all of these in a directory that is automatically cleaned by Maven, like `/target/`